src/clarity-angular/datagrid/datagrid.ts

```diff
@@ -176,18 +186,36 @@ export class Datagrid implements AfterContentInit, AfterViewInit, OnDestroy {
      * by querying the projected content. This is needed to keep track of the models currently
      * displayed, typically for selection.
      */
+
     @ContentChildren(DatagridRow) rows: QueryList<DatagridRow>;
 
+    /**
+     * We get deep nested DatagridActionOverflow children components, listen to the changes in them,
+     * and figure out if the datagrid has at least one actionable row.
+     */
+    @ContentChildren(DatagridActionOverflow, { descendants: true }) public actionableRows: QueryList<DatagridRow>;
+
     ngAfterContentInit() {
-        // TODO: Move all this to ngOnInit() once https://github.com/angular/angular/issues/12818 goes in.
-        // And when we do that, remove the manual step for each one.
+
+        this.actionableRows.changes.subscribe(() => {
```

Comment:
Sorry. The comment was lost during fixing the rebase conflicts. Also, I've added the subscribtion to this._subscriptions.